test/coverage/datamodel/UUriSerializerTest.cpp

```diff
@@ -23,16 +25,140 @@ class TestFixture : public testing::Test {
 
 	// Run once per execution of the test application.
 	// Used for setup of all tests. Has access to this instance.
-	TestFixture() = default;
-	~TestFixture() = default;
+	TestUUriSerializer() = default;
+	~TestUUriSerializer() = default;
 
 	// Run once per execution of the test application.
 	// Used only for global setup outside of tests.
 	static void SetUpTestSuite() {}
 	static void TearDownTestSuite() {}
 };
 
-// TODO replace
-TEST_F(TestFixture, SomeTestName) {}
+v1::UUri buildValidTestURI(const std::string& authority = "192.168.1.10") {
+	v1::UUri uri;
+	uri.set_authority_name(authority);
+	uri.set_ue_id(0x1100);
+	uri.set_ue_version_major(0xFF);
+	uri.set_resource_id(0x8000);
+	return uri;
+}
+
+// Positive test case - test serialization of UUri to string
+TEST_F(TestUUriSerializer, SerializeUUriToString) {
+	const auto& testUUri = buildValidTestURI();
```

Comment:
Can this be a const reference? The returned value is a temporary. I'm surprised this passes the linter. It should just be `auto`:
```suggestion
	auto testUUri = buildValidTestURI();
```

Reply:
Done